src/index.html

```diff
@@ -12,7 +12,53 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between the closing div tag of one stars block and the opening div tag of the next one.